client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -359,27 +364,26 @@ export const PatternLibrary = ( {
 					</PatternLibraryBody>
 				) }
 
-				{ isEnabled( 'readymade-templates/showcase' ) && isHomePage && <ReadymadeTemplates /> }
-
 				{ ! isEnabled( 'readymade-templates/showcase' ) && isHomePage && (
 					<PatternsCopyPasteInfo theme="dark" />
 				) }
```

Comment:
Similar to my comment above. We don't need to maintain a separate section for when the flag is disabled. Let's unify it. I think this will also remove the necessity of having separate `dark` and `gray` themes.

Reply:
On the new design we use a `gray` theme which we would need to keep.